.github/ISSUE_TEMPLATE/build.md

```diff
@@ -2,21 +2,46 @@
 name: Build
 about: This is a build issue for PLUTO that tracks dataloading, pluto improvements
   and additional info
-title: "[build] version: {e.g. 20v4}"
+title: "[build] version: {e.g. Major: 20v4, Minor: 22v3.1}"
 labels: 'data: ingestion'
 assignees:
 
 ---
 
 # Update version name
 
+### Major releases follow the naming convention YYvQ (where YY is the year, and Q is the Quarter) while minor releases follow the naming convention YYvQ.X (where YY is the year, Q is the quarter and X is the minor release version)
+
+### Major Release: To initiate a major release, you must change the VERSION and VERSION_PREV. All other variables can remain unchanged and will default to latest in the dataloading step. 
```

Comment:
is Q always gonna be the quarter or has it been more like "increment in the year"? like if 23v3 takes until November to release, would we change it to v4 and not release a v3?

also I wrote a little bit about this in the readme, but definitely think the instructions in the second part of this are good to have in this issue template!

Reply:
Good point